packages/core/src/SimpleTable/index.tsx

```diff
@@ -65,6 +67,13 @@ const TableHeaderCellContent = styled.th`
   font-weight: ${({ theme }) => theme.font.fontWeight.normal};
 
   user-select: none;
+
+  ${({ width }) =>
+    width !== undefined
+      ? css`
+          width: ${width}px;
```

Comment:
If at all possible, we should probably use relative widths. We use the simple table especially so the total width will be adjusted automatically without having to keep track of rendered sizes.

Reply:
If so, i'd change the name of the prop from `width` to `columWidthPercentage` or something. To make it more clear what it is.

Reply:
Changed to `columWidthPercentage`